src/amb_kf.c

```diff
@@ -244,9 +244,9 @@ void nkf_update(nkf_t *kf, double *measurements)
   DEBUG_EXIT();
 }
 
-void least_squares_solve_b(nkf_t *kf, const sdiff_t *sdiffs_with_ref_first,
-                      const double *dd_measurements, const double ref_ecef[3],
-                      double b[3])
+s8 least_squares_solve_b(nkf_t *kf, const sdiff_t *sdiffs_with_ref_first,
+                         const double *dd_measurements, const double ref_ecef[3],
+                         double b[3])
```

Comment:
This function is gone in my PR so maybe consider not modifying it here to ease merging?